test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -118,6 +118,7 @@
     "gasEstimateType": "none",
     "nonRPCGasFeeApisDisabled": "boolean"
   },
+  "JsonSnapsRegistry": "object",
```

Comment:
This seems unintentional given that `SnapsRegistry` is also in the snapshot? Can we just keep `SnapsRegistry`?